src/NumericalAlgorithms/DiscontinuousGalerkin/BoundarySchemes/StrongFirstOrder/StrongFirstOrderLocalTimeStepping.hpp

```diff
@@ -0,0 +1,103 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/BoundarySchemes/StrongFirstOrder/StrongFirstOrder.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Tags.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace dg {
+namespace BoundarySchemes {
+
+template <size_t Dim, typename VariablesTag, typename NumericalFluxComputerTag,
+          typename TemporalIdTag, typename TimeStepperTag>
+struct StrongFirstOrderLocalTimeStepping {
```

Comment:
This needs some documentation, even if that is basically just a link to the GTS version.